Resources/Private/BackendModule/src/Components/SubmitAndFetchNextButton.tsx

```diff
@@ -42,7 +42,7 @@ export default class SubmitAndFetchNextButton extends PureComponent {
                 className={'neos-button neos-button-success'}
                 disabled={isPersisting || isLoading || isGenerating || hasItemWithoutPropertyValue}>
```

Comment:
Du zeigst den Button an, während isLoading true is, ich vermute es wär logischer den hier zu verstecken!?